### PR TITLE
Update range slider and select component to be able to display a visu…

### PR DESCRIPTION
**Pull request by florianzaneray**
…ally hidden label element. This visually hidden label is needed for ADA compliance.

### CODE REVIEW

**Review by cbaxter713 — APPROVED** [review]
Looks good! Nice update

File: packages/vue/index/components/base/ZrRangeSlider.vue [hunk 1]
```diff
@@ -246,6 +265,10 @@
     height: 34px; /* thumb height + label font size */
     position: relative;
 
+    .visually-hidden {
```
Comment (tylerrzaneray) [inline packages/vue/index/components/base/ZrRangeSlider.vue:268]: `display: none;` will hide the label from assistive technologies and defeats the purpose of having it. Here's the visually hidden class we use on other sites:

```
.visually-hidden {
  position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  padding: 0;
  overflow: hidden;
  border: 0;
}
```

Comment (cbaxter713) [reply packages/vue/index/components/base/ZrRangeSlider.vue:268]: Great call @tylerrzaneray

Comment (tylerrzaneray) [reply packages/vue/index/components/base/ZrRangeSlider.vue:268]: Looks great!

**Review by tylerrzaneray — APPROVED** [review]

**Pull request merged**